### PR TITLE
Fix: Typos

### PR DESCRIPTION
**Pull request by Mithrandir0407**
## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the [code style][style-guide] of this project.
- [ ] I have updated the documentation accordingly.

[style-guide]: https://wiki.apache.org/jmeter/CodeStyleGuidelines

### CODE REVIEW

File: src/core/src/main/java/org/apache/jmeter/control/TransactionController.java [hunk 1]
```diff
@@ -263,7 +263,7 @@ public void triggerEndOfLoop() {
             // See Bug 56811
             // triggerEndOfLoop is called when error occurs to end Main Loop
             // in this case normal workflow doesn't happen, so we need
-            // to notify the childs of TransactionController and
+            // to notify the child of TransactionController and
```
Comment (ljacqu) [inline src/core/src/main/java/org/apache/jmeter/control/TransactionController.java]: Probably meant `children` here

Comment (Mithrandir0407) [reply src/core/src/main/java/org/apache/jmeter/control/TransactionController.java]: yes, you're right :)

File: src/core/src/main/java/org/apache/jmeter/threads/ThreadGroup.java [hunk 2]
```diff
@@ -361,7 +361,7 @@ public boolean stopThread(String threadName, boolean now) {
     }
 
     /**
-     * Hard Stop JMeterThread thrd and interrupt JVM Thread if interrupt is true
+     * Hard Stop JMeterThread third and interrupt JVM Thread if interrupt is true
```
Comment (FSchumacher) [inline src/core/src/main/java/org/apache/jmeter/threads/ThreadGroup.java]: This is probably `thread` and not `third`. We could probably get rid of `thr*d` here at all.

**Comment by FSchumacher** [comment]
Thanks for the pull request. It is always nice to know that someone looks at the code and cares enough to send in changes.

**Pull request closed without merging**